google/ggrc-core
# GGRC-8287 Global Admin can update Cycle via API PUT request

# Issue description

Global Admin can update Cycle via API PUT request

# Steps to test the changes

1. Sign in as Global Admin
2. Create Workflow object with Cycle
3. Send API PUT request with random Cycle title to change them
Actual: 200 code of response and title was changed
Expected: 403 code of response and title wasn't changed

# Solution description

reused WFCustomRestrictions mixin functionality

# Sanity checklist




# PR Review checklist
